VATSpy.dat

```diff
@@ -18424,19 +18424,20 @@ EHML|Bandbox CRC (Military)||EHAA
 EIDW|Dublin ACC (Up to FL245) - Dublin||EIDW
 EISN|Shannon ACC - Shannon||EISN
 EISN-L|Shannon ACC (Up to FL245) - Shannon|EISN_LS|EISN-L
-EKDK|Kobenhavn||EKDK
-EKDK-E|Kobenhavn (East)|EKDK_E|EKDK-E
-EKDK-E|Kobenhavn (East)|EKDK_D|EKDK-E
-EKDK-E|Kobenhavn (East)|EKDK_I|EKDK-E
+EKDK|Kobenhavn ACC||EKDK

```

Comment:
Suggest keep Kobenhavn as name.
Otherwise when a sector is offline it will display as "Kobenhavn ACC FIR"
It is logical to include ACC in areas where ACC name dosent match FIR name, for example in Sweden or Norway.


```suggestion
EKDK|Kobenhavn||EKDK
```